client/beefy/src/lib.rs

```diff
@@ -180,6 +152,24 @@ where
 	(import, voter_links, rpc_links)
 }
 
+/// BEEFY gadget network parameters.
+pub struct BeefyNetworkParams<B, N>
+where
+	B: Block,
+	N: GossipNetwork<B> + NetworkRequest + SyncOracle + Send + Sync + 'static,
```

Comment:
You should not bind traits at the type definition if they are not needed (they are not needed here). 

This makes downstream code most often easier to write.